src/app/App.js

```diff
@@ -72,10 +73,41 @@ export const App = ({ options }) => {
   });
 
   // Modal window
-  ModalWindow(app);
+  const modalWindow = ModalWindow(app);
   // Modal window methods
   // modalWindow.show(Here_some_content_as_HTMLElement);
   // modalWindow.close  to close the window and remove the content
+
+  const testButton = render({
+    component: Button({
+      id: 'abcd',
+      btnText: 'TEST BUTTON',
+      classList: [''],
+      onClickFn: testFunction,
+    }),
+    inside: app,
+  });
+
+  function testFunction() {
+    const contemodalWindowContent = ModalWindowContent(
```

Comment:
Proszę użyj tutal cammelCase'a =>  **conteModalWindowContent** :) 

I czy tutaj nie zgubiła się jakaś literka? Co to jest ten **conte**?

Reply:
a faktycznie, ale to i tak zniknie, bo jest tylko napisane, żeby przetestować :-)